templates/common/infra/bicep/core/host/functions-dotnet.bicep

```diff
@@ -0,0 +1,36 @@
+param environmentName string
```

Comment:
Look at the README.md file it is showing as a binary file for me


![image](https://user-images.githubusercontent.com/2163001/192825663-5dcb3e94-2e7e-4efa-9ef1-10acef617dce.png)



Please update the README title to : "ToDo Application with a C# API and Azure SQL DB on Static Web Apps and Functions"
![image](https://user-images.githubusercontent.com/2163001/192825715-a51ffec0-154b-45d8-a57a-2c245a45faf5.png)

Reply:
Resolved in latest commit.